portfolio/src/main/webapp/script.js

```diff
@@ -12,26 +12,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/*** Code Given in Template, not in use
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
-}
-*/
-
-
 /**
  * Fetches greeting from the server and adds it to the DOM.
  */
-function fetchAndShowGreeting() {
-  console.log('Fetching a greeting.');
+function fetchAndShowFavQuote() {
+  console.log('Fetching a favourite quote.');
```

Comment:
nitty nit: `quotes` :) same below

Reply:
Actually +1 to that :D I got a bit confused thinking if one quote will be shown or all three of them will be shown.

Reply:
Good point, thanks!